### PR TITLE
feat: push router performance metrics to prometheus

### PR DESCRIPTION
**Pull request by kimpers**
# Description

# Testing Instructions

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Test changes on the staging environment with [Matcha API Staging](https://api-staging.matcha.xyz).

    -   [ ] SRA/Limit orders
    -   [ ] Swap endpoints
    -   [ ] Meta transaction endpoints
    -   [ ] Depth charts

    For more information see `0x API Matcha smoke test runbook` in Quip.

### CODE REVIEW

**Comment by kimpers** [comment]
deploy staging

File: src/utils/sampler_metrics.ts [hunk 1]
```diff
@@ -17,6 +17,12 @@ const SAMPLER_BLOCK_NUMBER_GUAGE = new Gauge({
     help: 'Provides information about the gas limit detected during a sampler call',
```
Comment (fluffypill0w) [inline src/utils/sampler_metrics.ts:17]: Is there a reason for the const to be named `SAMPLER_BLOCK_NUMBER_GUAGE` instead of "SAMPLER_BLOCK_NUMBER_GAUGE"?

Comment (kimpers) [reply src/utils/sampler_metrics.ts:17]: probably no reason other than fat fingers 😂 , also mine should probably not prefixed with `SAMPLER_` 🙈

**Review by fluffypill0w — APPROVED** [review]

**Comment by kimpers** [comment]
deploy staging

**Comment by kimpers** [comment]
deploy production

**Review by dekz — APPROVED** [review]

**Pull request merged**